lib/http-api.js

```diff
@@ -97,8 +104,7 @@ HttpApi.prototype.request = function(request, callback) {
     }
     return self.getResponseBody(response);
   }, function(err) {
-    var responseTime = Date.now();
-    logger.debug("elappsed time : " + (responseTime - requestTime) + "msec");
+    onComplete();
```

Comment:
would upstream listeners be able to distinguish error response times from success response times?

Reply:
not currently; we're just looking for overall timing